x-pack/plugins/security_solution/public/management/cypress/e2e/mocked_data/reponse_actions_history.cy.ts

```diff
@@ -50,4 +50,25 @@ describe('Response actions history page', () => {
     cy.getByTestSubj('pagination-button-0').click();
     cy.getByTestSubj('response-actions-list-details-tray').should('exist');
   });
+
+  it('collapses expanded tray with a single click', () => {
+    loadPage(`/app/security/administration/response_actions_history`);
+    // 2nd row on 1st page
+    const row = cy.getByTestSubj('response-actions-list-expand-button').eq(1);
+
+    // expand the row
+    row.click();
+    cy.getByTestSubj('response-actions-list-details-tray').should('exist');
+    cy.url().should('include', 'withOutputs');
+
+    // collapse the row
+    cy.intercept('GET', '/api/endpoint/action*').as('getResponses');
+    row.click();
+    // wait for the API response to come back
+    // and then see if the tray is actually closed
+    cy.wait('@getResponses', { timeout: 500 }).then((xhr) => {
```

Comment:
why do we need `timeout` here?
also `xhr` in not used

Reply:
just to make sure we wait a bit extra long for the response to be back and, as the tray actually close (when the test passes) and then re-open quickly. Which is the actual bug. 

Remove the unused param ab7b001dafab47e8ee3508f3baab1b31aab1bb7a